src/metalnx-web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -54,9 +54,11 @@
 	<bean id="ep403"
 		class="com.emc.metalnx.handler.Http403ForbiddenEntryPoint" />
 
-	<sec:http auto-config="true" create-session="always"
+	<sec:http auto-config="true" create-session="always" use-expressions="false"
 		entry-point-ref="ep403">
 
+        <sec:csrf disabled="true"/>
```

Comment:
Please add a comment for this as well.

Reply:
This is for backward compatibility with spring security 3.  I added that in the comments.

We may want to enable this to protect against cross site request forgery in the future but that will take some rework.

Reply:
> This is for backward compatibility with spring security 3. I added that in the comments.

Which comments are you referring to? I may have missed them.

Reply:
https://github.com/irods-contrib/metalnx-web/blob/2933f579d3ed52adb08a46b48391eb3bee3d885c/src/metalnx-web/src/main/webapp/WEB-INF/applicationContext.xml#L61-L62

Are you seeing this change?

Reply:
Yep, I see them.